public/src/components/channelManagement/epicTests/epicTestVariantEditorCtasEditor.tsx

```diff
@@ -19,6 +19,11 @@ const DEFAULT_PRIMARY_CTA: Cta = {
   baseUrl: 'https://support.theguardian.com/contribute',
 };
 
+const DEFAULT_SECONDARY_CTA: Cta = {
```

Comment:
what happens if we don't have default values?

Reply:
Yeah we could leave them empty, then when a user selects `Custom` the fields would be empty by default, maybe that makes more sense?

Reply:
I think empty is preferable for 2nd cta as it could be anything! It should still error when they try to save

Reply:
Yeah that makes sense - I've made that change now!